### ISSUE TITLE
Alternate HTML/CSS for the CompanyInfo

### ISSUE THREAD

**Issue by rgoshko** [psadmin-io/io-styles-859#1]
Hi Dan, 

Here is the HTML/CSS I used for the CompanyInfo banner, this gives me a left and right floating DIV with the DBName and refresh time on the left and user name and ID on the right:

```
<div style="width: 100%; height: 17px; padding-top: 3px; padding-bottom: 3px;">
    <div id="ioDBName" style="float: left; margin-left: 7px;">:1 :2</div>
    <div id="ioWelcome" style="float: right; margin-right: 7px;">:3</div>
</div>
```

![2023-02-14_08-45-20](https://user-images.githubusercontent.com/49404109/218787204-224cbf29-3234-4767-99e4-8ed75946c6ec.png)

Thanks for the awesome code!

**Comment by iversond** [comment]
I like that Rob - that's a better way to write the HTML into the header. I'll update the docs shortly as I'm working on 8.60 themes right now.

**Comment by iversond** [comment]
I'm going to close this issue - the styles have been included as part of the 8.60 updates.
